README.md

```diff
@@ -2,7 +2,47 @@
 
 Automation of OpenShift 3 using [Ansible](http://www.ansible.com/)
 
-## **Note: This section is currently under active development.**
+## Provisioning Quickstart
+
+### Local Setup (one time, only)
+
+In addition to _cloning this repo_, you'll need the following:
+
+* Access to an OpenStack environment using an link:http://docs.openstack.org/user-guide/common/cli-set-environment-variables-using-openstack-rc.html[OpenStack RC File]
+  * File should be placed at `~/.config/openstack/openrc.sh`
+* A link:https://github.com/naturalis/openstack-docs/wiki/Howto:-Creating-and-using-OpenStack-SSH-keypairs-on-Linux-and-OSX[Key-pair created in OpenStack]
+* Docker installed (`yum install -y docker` on RHEL/Centos, `dnf install docker -y` on Fedora)
+  * If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+* An `~/.ansible.cfg` file containing the following:
+```
+[defaults]
+roles_path= /root/repository/rhc-ose/rhc-ose-ansible/roles:/root/repository/openshift-ansible/roles
+filter_plugins= /usr/share/ansible_plugins/filter_plugins:/root/repository/openshift-ansible/filter_plugins
+host_key_checking = False
+```
+* Clone this, and dependent `openshift-ansible` repositories into the same directory
+```
+cd ~/src/
+git clone git@github.com:redhat-cop/casl-ansible.git
+git clone git@github.com:openshift/openshift-ansible.git
+```
+* Copy `casl-ansible/inventory/sample.casl.example.com.d/inventory/clouds.yaml` to `~/.config/openstack/clouds.yaml`
+
+Cool! Now you're ready to provision OpenShift clusters
+
+### Provision a Cluster
+
+To start, we'll provision a the `sample.casl.example.com` cluster defined in the `casl-ansible/inventory` directory.
+
+1. Edit `casl-ansible/inventory/sample.casl.example.com/hosts` and edit the `# OpenStack Provisioning Variables` and `# Subscription Management` sections at the top to match your environment/project/tenant.
```

Comment:
Do we want to include any additional info re: additional parameters that can/should be adjusted?